index.js

```diff
@@ -1,26 +1,20 @@
 const express = require('express')
-const carData = require('./cars')
+const { getAllVehicles, getAllVehiclesByModel } = require('./controllers/vehicles')
+const { getAllModelNames, getModelById } = require('./controllers/models')
+const { getAllModelYears, getModelYearByVehicles } = require('./controllers/modelyears')
 
 const app = express()
 
-app.set('view engine', 'pug')
-app.use(express.static('public'))
+app.get('/vehicles', getAllVehicles)
+app.get('/vehicles/:identifier', getAllVehiclesByModel)
 
-// app.get('/cars', (_request, response) => {
-//   return response.send(carData)
-// })
+app.get('/models', getAllModelNames)
+app.get('/models/:id', getModelById)
 
-app.get('/cars', (_request, response) => {
-  return response.render('index', { carData })
-})
-
-
-
-app.all('*', (_request, response) => {
-  return response.status(404).send('sorry! page not found')
-})
+app.get('/modelyears', getAllModelYears)
+app.get('/modelyears/:identifier', getModelYearByVehicles)
```

Comment:
You removed your documentation, without it there is no way for me to know how to call your API correctly.